.github/workflows/docker-unstable.yml

```diff
@@ -6,31 +6,37 @@ on:
       - master
 
 jobs:
-
-  push:
-
+  docker-build-and-push:
     runs-on: ubuntu-latest
-
     strategy:
       fail-fast: false
       matrix:
         dockerfile:
           - {"name": etna-cpu, "path": docker/etna-cpu/Dockerfile}
           - {"name": etna-cuda-11.6.2, "path": docker/etna-cuda-11.6.2/Dockerfile}
-
     steps:
       - uses: actions/checkout@v2
-
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```

Comment:
We should plan migration to 3.10 python everywhere

Reply:
Should I set 3.10 everywhere in this task?

Reply:
Better in other PR